app/src/main/java/com/morlunk/mumbleclient/Settings.java

```diff
@@ -164,6 +164,9 @@
     public static final String PREF_SHOW_USER_COUNT = "show_user_count";
     public static final boolean DEFAULT_SHOW_USER_COUNT = false;
 
+    public static final String PREF_START_UP_IN_PINNED_MODE = "startUpInPinnedMode";
+    public static final boolean DEFAULT_START_UP_IN_PINNED_MODE = true;
```

Comment:
This should probably be false by default. I suspect that regular users of Plumble will still want to see all channels by default, lest they be met with an empty screen when connecting to a new server.